tests/unittests/CoreGraphics.drawing/ImageComparison.h

```diff
@@ -22,6 +22,8 @@
 enum class ImageComparisonResult : unsigned int { Unknown = 0, Incomparable, Different, Same };
 
 struct Pixel {
+    // Value at which pixel differences become perceptible
```

Comment:
I'm pretty sure that this isn't the actual value for just-noticeable-difference, just most likely under it. Could you change the comment to reflect that?